lib/rubocop/cop/chef/deprecation/recipe_metadata.rb

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright:: 2019, Chef Software Inc.
+# Author:: Tim Smith (<tsmith@chef.io>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module RuboCop
+  module Cop
+    module Chef
+      module ChefDeprecations
+        # The recipe metadata.rb method is not used and is unnecessary in cookbooks. Recipes should be documented
+        # in the README.md file instead.
+        #
+        # @example
+        #
+        #   # bad
```

Comment:
```suggestion
        #   # unnecessary
```

Reply:
I just noticed that this is the way we mark this as /goog/ vs /bad/ - It felt a bit too much to mark it as bad but if that is the pattern just ignore me 😅

Reply:
That's the standard rubocop comment block. We could potentially change it up, but we should do it all at once.